quest_test/test_manager.py

```diff
@@ -110,7 +112,8 @@ async def workflow(arg: int):
 
 @pytest.mark.asyncio
 async def test_manager_background():
-    storage = InMemoryBlobStorage()
+    test_state = Path('test-state')
+    storage = LocalFileSystemBlobStorage(test_state)
```

Comment:
Let's fix the deep copy in InMemory so we can use those.

Reply:
Done